modules/ml/src/main/java/org/apache/ignite/ml/math/impls/storage/matrix/SparseDistributedMatrixStorage.java

```diff
@@ -115,6 +115,9 @@ public SparseDistributedMatrixStorage(int rows, int cols, int stoMode, int acsMo
         // Cache is partitioned.
         cfg.setCacheMode(CacheMode.PARTITIONED);
 
+        // Otherwise we can get null from get after put.
+        cfg.setReadFromBackup(false);
```

Comment:
Please describe this change

The problem is that we have a lot of Java copy-paste configs for many primitives and in 1.0 release it should have the unified configuration to avoid strange cluster effects

Do you have a test for this case?